Client/src/main/java/org/projectcardboard/client/view/battleview/SpellAnimation.java

```diff
@@ -30,7 +32,13 @@
   private Group mapGroup;
 
   SpellAnimation(Group mapGroup, String fileName, double x, double y) throws Exception {
+    boolean isCustomCardFlag = false;
     InputStream plistR = this.getClass().getResourceAsStream("/fx/" + fileName + ".plist.json");
+    if (plistR == null) {
+      isCustomCardFlag = true;
+      String customCardPath = Config.getInstance().getCustomCardSpritesPath().toString() + "/";
+      plistR = new FileInputStream(customCardPath + fileName + ".plist.json");
```

Comment:
maybe add check... if path exists...

Reply:
the path will exist if the config file exists, so i think a check is not needed